specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/CommonDefinitions.json

```diff
@@ -1185,6 +1185,204 @@
+          "description": "Timeout, in seconds, for a request to respond",
+          "format": "int32",
+          "type": "integer"
+        },
+        "connectionTimeoutInSeconds": {
+          "description": "Timeout, in seconds, for a request to initiate a connection",
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "HttpRetryPolicy": {
+      "description": "Policy that defines http request retry conditions",
+      "type": "object",
+      "properties": {
+        "maxRetries": {
+          "description": "Maximum number of times a request will retry",
+          "format": "int32",
+          "type": "integer"
+        },
+        "retryBackOff": {
+          "description": "Settings for retry backoff characteristics",
+          "type": "object",
+          "properties": {
+            "initialDelayInMilliseconds": {
+              "description": "Initial delay, in milliseconds, before retrying a request",
+              "format": "int64",
+              "type": "integer"
+            },
+            "maxIntervalInMilliseconds": {
+              "description": "Maximum interval, in milliseconds, between retries",
+              "format": "int64",
+              "type": "integer"
+            }
+          }
+        },
+        "matches": {
+          "description": "Conditions that must be met for a request to be retried",
+          "type": "object",
+          "properties": {
+            "headers": {
+              "description": "Headers that must be present for a request to be retried",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HeaderMatch"
+              }
+            },
+            "httpStatusCodes": {
+              "description": "Additional http status codes that can trigger a retry",
+              "type": "array",
+              "items": {
+                "format": "int32",
+                "type": "integer"
+              }
+            },
+            "errors": {
+              "description": "Errors that can trigger a retry",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "TcpRetryPolicy": {
+      "description": "Policy that defines tcp request retry conditions",
+      "type": "object",
+      "properties": {
+        "maxConnectAttempts": {
+          "description": "Maximum number of attempts to connect to the tcp service",
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "circuitBreakerPolicy": {
+      "description": "Policy that defines circuit breaker conditions",
+      "type": "object",
+      "properties": {
+        "consecutiveErrors": {
+          "description": "Number of consecutive errors before the circuit breaker opens",
+          "format": "int32",
+          "type": "integer"
+        },
+        "intervalInSeconds": {
```

Comment:
Envoy continuously checks endpoints in the background to either eject in case of failure or return to service after resuming normal operation. Can we update the description so it's clear that this can result in both new ejections as well as hosts being returned to service? Also, let's add the default interval in the description (10 seconds in Envoy).